src/System.Private.ServiceModel/src/System/ServiceModel/Security/SecurityAppliedMessage.cs

```diff
@@ -0,0 +1,471 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime;
+using System.Security.Cryptography;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Security.Tokens;
+using System.Xml;
+using System.IdentityModel.Tokens;
+using System.Collections.Generic;
+using IPrefixGenerator = System.IdentityModel.IPrefixGenerator;
+using ISecurityElement = System.IdentityModel.ISecurityElement;
+// using XmlAttributeHolder = System.IdentityModel.XmlAttributeHolder;
```

Comment:
Great catch.  I'm removing this line because XmlAttributeHolder in ServiceModel exists and appears to be identical to the IdentityModel one

---
In reply to: [78405348](https://github.com/dotnet/wcf/pull/1515#discussion_r78405348) [](ancestors = 78405348)